src/SFA.DAS.Reservations.Web.UnitTests/Services/WhenGettingStartDates.cs

```diff
@@ -33,16 +34,31 @@ public class WhenGettingStartDates
             StartDateService startDateService)
         {
             var now = mockCurrentDateTime.Object.Now;
-            var expectedDates = new List<DateTime>();
+            var expectedDates = new List<StartDateModel>();
             for (var i = 0; i < 6; i++)
             {
-                expectedDates.Add(now.AddMonths(i).AddDays(1-now.Day).Date);
+                var startDateModel = BuildStartDateModel(now.AddMonths(i));
+                expectedDates.Add(startDateModel);
             }
 
             var dates = await startDateService.GetStartDates();
 
             dates.Count().Should().Be(6);
             dates.Should().BeEquivalentTo(expectedDates);
         }
+
+        private StartDateModel BuildStartDateModel(DateTime now)
```

Comment:
I think this is used elsewhere in a different test class, might be worth pulling it out into some common ObjectMother builder

Reply:
after discussion will pick this up later

Reply:
deferring to future story